docs/readmes/lte/suci_extensions.md

```diff
@@ -0,0 +1,92 @@
+
+## Detailed Design
+
+![Design of SUCI Extensions Network Flow](assets/lte/design_of_suci_extensions.png?raw=true "Design of SUCI Extensions Network Flow")
+
+The diagram above represents the network create/update flow and the modules involved in processing the NMS request. Also represents how the SuciProfiles are fetched from Orc8r.
+
+### Orc8r
+
+- To facilitate REST API updates, the LTE Network data model will accept a list of SuciProfiles and obsidian handlers.
+- Subscriberdb RPC servicer to handle the gRPC request from AGW, which populates the list of SuciProfiles per network.
+
+### AGW
+
+- AMF to invoke SIDF Module of subscriberdb for SUCI decryption.  In initial registration NAS message, if type of identity is SUCI (1) and protection scheme Id is either ECIES scheme profile A OR ECIES scheme profile B, then AMF initiates GRPC call to subscriberdb with parameters such as Home network public key identifier, ue ciphertext and encrypted imsi.
+- SubscriberDB periodically fetches the list of SuciProfiles from Orc8r and stores it as in- memory data.
+- SubscriberDB has SIDF functionality, that performs an in-memory data lookup to extract the private key associated with the home network public key identifier and protection scheme that we obtained from the initial registration NAS message. SIDF has an algorithm that uses the private key to decode the SUCI.
+
+## Prerequisites
+
+Prior to configuring SUCI extensions, you must first support integrated 5G SA FWA.
+
+## High Level Call Flow
+
+The call flow between AMF and SubscriberDB in magma for SUCI extensions is shown in the figure below.
+
+```mermaid
+sequenceDiagram
+    MME(AMF)->>Subscriberdb: Registration Request with type of identity is SUCI and Protection scheme either Profile A or Profile B 
+    Note over MME(AMF),Subscriberdb: Public Key Identifier, UE cypertext and Encryped IMSI
+    Subscriberdb->>MME(AMF): Decrypted IMSI
+```
+
+## Configuration SUCI Extensions
+
+While creating the LTE network, SUCI Profiles can be established using the below swagger API.
+
+Swagger API : POST - /lte
+
+Below is the payload.
+
+![Create New LTE Network with SuciProfile](assets/lte/create_new_lte_network_with_suciprofile.png?raw=true "Create New LTE Network with SuciProfile")
+
+If we want to add a SUCI Profile to an existing LTE network, we may do it with the swagger API listed below.
+
+Swagger API : PUT - /lte/{network_id}
+
+Below is the payload.
+
+![Update Existing LTE Network with SuciProfile](assets/lte/update_existing_lte_network_with_suciprofile.png?raw=true "Update Existing LTE Network with SuciProfile")
+
+## 3GPP References
+
+- 3GPP TS 23.003 version 15.4.0 Release 15 - Digital cellular telecommunications system (Phase 2+) (GSM); Universal Mobile Telecommunications System (UMTS); Numbering, addressing and identification
+- 3GPP TS 33.501 version 15.2.0 Release 15 - 5G; Security architecture and procedures for 5G System

```

Comment:
Can we also give a link, where those can be found, please.

Reply:
Addressed Review Comment